### PR TITLE
Reformat CSS (Hopefully doesn't break anything this time)

### PR DESCRIPTION
**Pull request by JaredTamana**

### CODE REVIEW

File: _sass/jekyll-theme-hacker.scss [hunk 1]
```diff
@@ -1,764 +1,749 @@
...
-    background-color: var(--bodybkg);
-    /*transition: all 300ms;*/
-}
-
-a {
-    color: #1395e0;
-    text-decoration: none;
-}
-
-.clearfix {
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss:68]: We need to keep clearfix, just means we dont have to keep remaking more and more elements with the same stuff.

File: _sass/jekyll-theme-hacker.scss [hunk 2]
```diff
@@ -1,764 +1,749 @@
...
-.contributor-avatar {
-    object-fit: contain;
-    width: 50%;
-}
-
-.contributor-wrapper {
-    display: flex;
-}
-
-.hide {
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss:90]: Keep hide rule. Used for when injecting JS and other shorts, and also keeps from making more elements

File: _sass/jekyll-theme-hacker.scss [hunk 3]
```diff
@@ -1,764 +1,749 @@
...
+    opacity: 0.5;
+    max-width: 1000px;
+    text-align: end;
+    margin: auto;
+    padding: 0 0 15px;
+}
+
+.footersubtext {
+    display: block;
+    -webkit-margin-before: 1em;
```
Comment (TitusStudiosMediaGroup) [inline _sass/jekyll-theme-hacker.scss:153]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 4]
```diff
@@ -1,764 +1,749 @@
...
+            margin-block-start: 1em;
+    -webkit-margin-after: 1em;
```
Comment (TitusStudiosMediaGroup) [inline _sass/jekyll-theme-hacker.scss:155]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 5]
```diff
@@ -1,764 +1,749 @@
...
+            margin-block-end: 1em;
+    -webkit-margin-start: 0;
```
Comment (TitusStudiosMediaGroup) [inline _sass/jekyll-theme-hacker.scss:157]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 6]
```diff
@@ -1,764 +1,749 @@
...
+            margin-inline-start: 0;
+    -webkit-margin-end: 0;
```
Comment (TitusStudiosMediaGroup) [inline _sass/jekyll-theme-hacker.scss:159]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 7]
```diff
@@ -1,764 +1,749 @@
...
+    font-weight: 100;
+    color: var(--footer-text);
+    opacity: 0.5;
+    max-width: 1000px;
+    text-align: end;
+    margin: auto;
+    padding: 0 0 15px;
+}
+
+.footersubtext {
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss:190]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 8]
```diff
@@ -1,764 +1,749 @@
...
+    opacity: 0.5;
+    font-size: 11px;
+    float: right;
+}
+
+.container {
+    margin: 0 35px;
+}
+
+#main_content {
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss:266]: Added unwanted properties

File: _sass/jekyll-theme-hacker.scss [hunk 9]
```diff
@@ -1,764 +1,749 @@
...
+    max-width: 1000px;
+    padding-left: 5px;
+    padding-right: 20px;
+    margin: auto;
+}
+
+.navlogo {
+    width: 50px;
+    height: 50px;
+    background-image: url(https://protogencollection.titusstudios.net/data/static/images/icon-whitecenter.png);
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss]: Should always encase in quotations

File: _sass/jekyll-theme-hacker.scss [hunk 10]
```diff
@@ -1,764 +1,749 @@
...
+
+@-webkit-keyframes fadein-welcome {
+    from {
+        opacity: 0;
+        top: -20px
+    }
+
+    to {
+        opacity: 1;
+        top: 0px;
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss:721]: I dont think this is meant to be here.

File: _sass/jekyll-theme-hacker.scss [hunk 11]
```diff
@@ -1,764 +1,749 @@
...
+    .navquicklinks {
+        display: none;
+    }
+
+    .navmobilebutton {
+        display: block;
+    }
+}
+
+@media screen and (max-width 460px) {
```
Comment (aerofur) [inline _sass/jekyll-theme-hacker.scss]: Aye bro wheres my colon

**Review by aerofur — COMMENTED** [review]
Most of this is very good. However there are also issues with it. Please have a look at some of the comments I have put on it, and run this on your local to test if everything is working as intended (reference active website for this).

**Comment by aerofur** [comment]
I may have missed some, which is why I ask. Since everything goes in alphabetical order its hard to tell if its the same. Cheers

File: _sass/jekyll-theme-hacker.scss (hunk 7 above)
Comment (JaredTamana) [reply _sass/jekyll-theme-hacker.scss:190]: Not sure what you're referring to here, but these prefixes are needed for compatibility reasons.

File: _sass/jekyll-theme-hacker.scss (hunk 8 above)
Comment (JaredTamana) [reply _sass/jekyll-theme-hacker.scss:266]: Again, not sure which properties you're referring to.

File: _sass/jekyll-theme-hacker.scss (hunk 10 above)
Comment (JaredTamana) [reply _sass/jekyll-theme-hacker.scss:721]: Not sure what you mean by that

**Comment by JaredTamana** [comment]
Some changes made. Some of your requests were unclear, please clarify.

**Comment by aerofur** [comment]
Hi. Sorry about that, for some reason github wasnt showing the diff for the new code which is why all my comments make no sense. Ill take a look now and see.

File: _sass/jekyll-theme-hacker.scss (hunk 9 above)
Comment (JaredTamana) [reply _sass/jekyll-theme-hacker.scss]: Technically the beautifier is correct in this case, quotations aren't strictly necessary, but I'll add them anyways

**Comment by aerofur** [comment]
Current version appers to be valid. Merging.

**Pull request merged**